downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md

```diff
@@ -47,105 +42,101 @@ A few backward-incompatible changes have been introduced during the Swan Lake Be
 
 Annotations are now supported on tuple members. Tuple members can be annotated using annotations declared with the `field` attachment point.
 
-```ballerina
-annotation annot on field;
+  ```ballerina
+  annotation annot on field;
 
-type T [int, @annot string];
-```
+  type T [int, @annot
+      string ];
+  ```
 
-Annotations are not allowed on the tuple rest descriptor. 
-
-### Improvements
+  Annotations are not allowed on the tuple rest descriptor. 
 
 ### Bug fixes
 
-- Annotation values of fields of record type descriptors that are not defined with a type definition are now accessible at runtime
+- Annotation values of fields of record type descriptors that are not defined with a type definition are now accessible at runtime.
 
-To view bug fixes, see the [GitHub milestone for Swan Lake 2201.4.0](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+label%3ATeam%2FCompilerFE+milestone%3A2201.4.0+is%3Aclosed+label%3AType%2FBug).
+To view other bug fixes, see the [GitHub milestone for Swan Lake 2201.4.0](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+label%3ATeam%2FCompilerFE+milestone%3A2201.4.0+is%3Aclosed+label%3AType%2FBug).
 
 ## Runtime updates
 
 ### New features
 
 #### Get the strand dump during `bal test`
 
-When running the tests in a Ballerina package or a file using the `bal test` command, the strand dump can be obtained by sending the `SIGTRAP` signal to that process.
+- When running the tests in a Ballerina package or a file using the `bal test` command, the strand dump can be obtained by sending the `SIGTRAP` signal to that process.
```

Comment:
Please recheck all bullet points. (Generally) we don't use bullet points if there is a heading for the new feature/improvement.

Reply:
Shall we go with the guideline that we add bullets only for a list and if there is only one point, we add it without a bullet point. @anupama-pathirage @MaryamZi

Reply:
+1 I think we don't need bullet point if it is only one item in the list